### ISSUE TITLE
"no such file or directory" when creating model during the "creating adapter layer" step

### ISSUE THREAD

**Issue by meow-d** [jmorganca/ollama#1274]
when i run `ollama create storywriter`, i get:
```
transferring model data
reading model metadata
creating template layer
creating system layer
creating adapter layer
Error: open /@sha256:439bdfbd08b0143c5f5f97154d76676a5348a5a00a2fac38fdc8d1c4498d67d3: no such file or directory
```

btw i'm running on Fedora 39

my Modelfile, just in case:
```
FROM llama2-uncensored:latest

TEMPLATE """{{ .System }}

### HUMAN:
{{ .Prompt }}

### RESPONSE:
"""

PARAMETER stop "### HUMAN:"
PARAMETER stop "### RESPONSE:"

SYSTEM """
"""

ADAPTER ./adapter_model.bin
```

**Comment by technovangelist** [comment]
And llama2-uncensored:latest is on your machine and the adapter file is in the same place where you are running that command from?

**Comment by meow-d** [comment]
no, llama2-uncensored:latest is downloaded using `ollama pull`

**Comment by meow-d** [comment]
thank you. but the problem is still there after i updated.

**Comment by mhunjaai** [comment]
I got the same problem @BruceMacD on both linux + mac.
In v0.1.9 the problem does not occur on creating the model based on the Modelfile with local adapter path.
However in v0.1.10 till latest it also fails with the error code from above. Maybe this helps.

**Comment by solablue** [comment]
I am facing the same issue. 

```
transferring context    
reading model metadata    
⠋ creating adapter layer  Error: open /@sha256:ff1527d49453147c6bd4a89ac61c8cb3948aea7d1787b2540330b5df7335e0ba: no such file or directory

```

**Comment by technovangelist** [comment]
Are folks still experiencing this issue? We are now on 0.1.17 so wondering if it has been solved. If not, perhaps we can get a copy of an adapter that is causing problems, and we can try to recreate the issue. I'm not sure what the right way to send that file over is, so ping here and we can coordinate. Thanks @solablue @meow-d @mhunjaai for being a great part of this remarkable community.

**Comment by PhilipAmadasun** [comment]
@technovangelist  I'm having the same "file_not_found" problems. Here is a link to my `adapter_model.bin` [here](https://huggingface.co/uyiosa/test_falcon_7b_model/tree/main)

**Comment by mxyng** [comment]
This should be fixed now (at least >=0.1.20 but maybe slightly older) where creating a model (including adapters!) should work with remote servers using local files. Please reopen the issue if that's not the case
